cmd/migrate.go

```diff
@@ -116,26 +116,33 @@ func runMigration(_ *cobra.Command, _ []string) error {
 
 	goose.SetBaseFS(assets.EmbedMigrations)
 
-	if version > 0 {
+	if targetVersion > 0 {
```

Comment:
Can we add the shortest case first? And make the logs more consistent

E.g. always print the current version and always print what migrations are about to be run and when they are done, e.g.

```go
currentVersion, err := goose.GetDBVersion(db)
if err != nil {
	log.Fatal(err)
}
if targetVersion == 0 {
  log.Println("current version %d, running all migrations")
...
} else {
  log.Printf("current version %d, migrating to %d", currentVersion, targetVersion)
  targetInt64Version := int64(targetVersion)
  if targetInt64Version == currentVersion {
    log.Println("nothing to do")
  } else ...
  ...
  ...
}
```
log.Println("migration done")